### PR TITLE
zh-trans: Add docs/contribute/_index.md

### PR DESCRIPTION
**Pull request by liyuan198251**
Signed-off-by: 李元10098251 <li.yuan4@zte.com.cn>

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.13 Features: set Milestone to 1.13 and Base Branch to dev-1.13
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/contribute/_index.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+你对 Kubernetes 文档可以做出的贡献不仅限于列表列出的条目，但它可以帮助你开动起来。
+
+<!--
+- [Anyone](/docs/contribute/start/)
+  - File actionable bugs
+-->
+
+- [任何人](/docs/contribute/start/)
+  - 修改操作类错误
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md]: 登记记录可以修正的错误

File: content/zh/docs/contribute/_index.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+  - Review PRs
+  - Create diagrams, graphics assets, and embeddable screencasts / videos
+  - Localization
+  - Contribute to other repos as a docs representative
+  - Edit user-facing strings in code
+  - Improve code comments, Godoc
+-->
+
+- [评审者](/docs/contribute/intermediate/)
+  - 提出新特性
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md]: 为新功能特性编写文档

File: content/zh/docs/contribute/_index.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+  - 对 issue 进行筛选和分类
+  - 评审 PR
+  - 创建图表、图形分析和嵌入式的屏幕/视频
+  - 本地化
+  - 作为文档持有人为其他项目仓库做贡献
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md]: 作为 docs 小组的代表 ...

File: content/zh/docs/contribute/_index.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+  - Publish contributor content by approving and merging PRs
+  - Participate in a Kubernetes release team as a docs representative
+  - Propose improvements to the style guide
+  - Propose improvements to docs tests
+  - Propose improvements to the Kubernetes website or other tooling
+-->
+
+- [审批者](/docs/contribute/advanced/)
+  - 通过批准和合并 PR 发布贡献成果
+  - 作为文档持有人参与 Kubernetes 发布团队
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md]: 作为 docs 团队的代表 ...

File: content/zh/docs/contribute/_index.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+  commitment to the project and is granted the ability to merge pull requests
+  and thus to publish content on behalf of the Kubernetes organization.
+  Approvers can also represent SIG Docs in the larger Kubernetes community.
+  Some of the duties of a SIG Docs approver, such as coordinating a release,
+  require a significant time commitment.
+-->
+
+- SIG 文档_审批者_是对项目持续贡献，并被授予合并 PR 权限和代表 Kubernetes 组织发布内容的成员。
+  批准人也可以在大型的 Kubernetes 社区中提出 SIG 文档。
+  SIG 文档审批者的一些职责，如协调发布版本，需要一个重要的时间承诺。
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md]: > a significant time commitment.

大量的时间投入

File: content/zh/docs/contribute/_index.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+- A SIG Docs _approver_ is a member in good standing who has shown a continued
+  commitment to the project and is granted the ability to merge pull requests
+  and thus to publish content on behalf of the Kubernetes organization.
+  Approvers can also represent SIG Docs in the larger Kubernetes community.
+  Some of the duties of a SIG Docs approver, such as coordinating a release,
+  require a significant time commitment.
+-->
+
+- SIG 文档_审批者_是对项目持续贡献，并被授予合并 PR 权限和代表 Kubernetes 组织发布内容的成员。
+  批准人也可以在大型的 Kubernetes 社区中提出 SIG 文档。
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md]:
```suggestion
  批准人也可以在更广泛的 Kubernetes 社区中代表 SIG Docs 团队。
```

File: content/zh/docs/contribute/_index.md [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+{{% capture overview %}}
+
+<!--
+If you would like to help contribute to the Kubernetes documentation or website,
+we're happy to have your help! Anyone can contribute, whether you're new to the
+project or you've been around a long time, and whether you self-identify as a
+developer, an end user, or someone who just can't stand seeing typos.
+-->
+
+如果您想帮助对 Kubernetes 文档或网站做出贡献，我们很高兴得到你的帮助！
```
Comment (xiangpengzhao) [inline content/zh/docs/contribute/_index.md]: 如果用“您”，则全文都统一起来。请检查全文。

File: content/zh/docs/contribute/_index.md [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
+任何人都可以做出贡献，无论你刚参与项目还是参与了很长时间，无论你是开发人员还是用户，或是无法忍受看到拼写错误的人。
+
+<!--
+For more ways to get involved in the Kubernetes community or to learn about us,
+also visit the [Kubernetes community site](/community/).
+-->
+
+更多途经参与 Kubernetes 社区或了解我们，请访问 [Kubernetes 社区网站](/community/)。
```
Comment (xiangpengzhao) [inline content/zh/docs/contribute/_index.md]: s/途经/途径

File: content/zh/docs/contribute/_index.md [hunk 9]
```diff
@@ -0,0 +1,173 @@
...
+  参见 [社区成员](https://github.com/kubernetes/community/blob/master/community-membership.md) 中对于成员资格的具体标准。
+
+<!--
+- A SIG Docs _reviewer_ is a member of the Kubernetes organization who has
+  expressed interest in reviewing documentation pull requests and who has been
+  added to the appropriate Github group and `OWNERS` files in the Github
+  repository, by a SIG Docs Approver.
+-->
+
+- SIG 文档_评审者_是对评审文档 PR 感兴趣，并被 SIG 文档审批者添加到 Github 组群并在 Github 仓库中 `OWNERS` 文件的 Kubernetes 组织的成员。
```
Comment (xiangpengzhao) [inline content/zh/docs/contribute/_index.md]: s/组群/群组

File: content/zh/docs/contribute/_index.md [hunk 10]
```diff
@@ -0,0 +1,173 @@
...
+
+- SIG 文档_审批者_是对项目持续贡献，并被授予合并 PR 权限和代表 Kubernetes 组织发布内容的成员。
+  批准人也可以在大型的 Kubernetes 社区中提出 SIG 文档。
+  SIG 文档审批者的一些职责，如协调发布版本，需要一个重要的时间承诺。
+
+<!--
+## Ways to contribute
+-->
+
+## 贡献途经
```
Comment (xiangpengzhao) [inline content/zh/docs/contribute/_index.md]: s/途经/途径

**Comment by liyuan198251** [comment]
已更新，非常非常感谢两位的指导

**Pull request merged**